test/dynamo/test_misc.py

```diff
@@ -925,6 +925,314 @@ def forward(self, arg0_1: "f32[3][1]cpu", arg1_1: "f32[3][1]cpu", arg2_1: "f32[3
             cleanup_op("mylib::foo")
             del lib
 
+    def run_aot_eager(self, f, orig_args):
```

Comment:
In the trivial case (where we are mutating a Tensor instead of the view), can we get auto_functionalized to look better? That is, we shouldn't need to store x_size x_stride x_storage_offset in this case. This happens in the common case and would help convince me that the code works the same the old auto_functionalized.

Reply:
you marked resolved, by mistake or still concern?
I do avoid generates as_strided for that case when in the decompose. but I keep the meta data in the auto_funcitonalize for simplicity. 

 Idk this can create issues down the road with how other passes interacts with auto_funcitonalize semantics.